docs/src/main/paradox/index.md

```diff
@@ -47,7 +47,6 @@ The [Alpakka project](https://doc.akka.io/docs/alpakka/current/) is an open sour
 * [InfluxDB](influxdb.md)
 * [IronMQ](ironmq.md)
 * [JMS](jms/index.md)
-* [MapR Database](external/mapr.md)
```

Comment:
"In August 2019, following financial difficulties, the technology and intellectual property of the company were sold to Hewlett Packard Enterprise."
https://en.wikipedia.org/wiki/MapR